pvactools/tools/pvacview/server.R

```diff
@@ -967,7 +967,12 @@ server <- shinyServer(function(input, output, session) {
     if (is.null(df$mainTable)) {
       return()
     }
-    data <- df$mainTable[, !(colnames(df$mainTable) == "Evaluation") & !(colnames(df$mainTable) == "Eval") & !(colnames(df$mainTable) == "Select") & !(colnames(df$mainTable) == "Scaled BA") & !(colnames(df$mainTable) == "Scaled percentile") & !(colnames(df$mainTable) == "Tier Count") & !(colnames(df$mainTable) == "Comments") & !(colnames(df$mainTable) == "Gene of Interest") & !(colnames(df$mainTable) == "Allele") & !(colnames(df$mainTable) == "Bad TSL")]
+    data <- df$mainTable[, !(colnames(df$mainTable) == "Evaluation") & !(colnames(df$mainTable) == "Eval") & !(colnames(df$mainTable) == "Select")
+                         & !(colnames(df$mainTable) == "Scaled BA") & !(colnames(df$mainTable) == "Scaled percentile") & !(colnames(df$mainTable) == "Tier Count")
+                         & !(colnames(df$mainTable) == "Comments") & !(colnames(df$mainTable) == "Gene of Interest") & !(colnames(df$mainTable) == "Bad TSL")
+                         & !(colnames(df$mainTable) == "Col RNA Expr") & !(colnames(df$mainTable) == "Col RNA VAF") & !(colnames(df$mainTable) == "Col Allele Expr")
+                         & !(colnames(df$mainTable) == "Col RNA Depth") & !(colnames(df$mainTable) == "Col DNA VAF") & !(colnames(df$mainTable) == "Percentile Fail")
+                         & !(colnames(df$mainTable) == "Has Prob Pos")]
```

Comment:
Could this be more concisely written like this?

colsToDrop = colnames(df$mainTable) %in% c("Evaluation", "Eval", "Select", "Scaled BA") #etc
data <- df$mainTable[,!(colsToDrop)]

Reply:
Yes great idea, I'll change it accordingly.